tests/api/runtime_handlers/test_sparkjob.py

```diff
@@ -170,21 +160,9 @@ async def test_delete_resources_completed_crd(
             self.runtime_handler,
             len(list_namespaced_crds_calls),
         )
-        self._assert_list_namespaced_pods_calls(
```

Comment:
what about the:
```
            [self.executor_pod, self.driver_pod],
            # additional time for wait for pods deletion - simulate pods gone
            [],
```
should be asserted no?

Reply:
I removed it while I was debugging some stuff but brought it back now.